environment.yml

```diff
@@ -0,0 +1 @@
+environment-pinned-linux.yml
```

Comment:
I'm not sure what the purpose of this file is? Not mentioned in new docs (as far as I can see)